float32 kick_speed
---
